pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pysus"
-version = "0.6.1"
+version = "0.6.2"
```

Comment:
I've forgot to change this version in the PR that was released as 0.6.2, do I have to set to 0.6.3 before merging?